main.tex

```diff
@@ -515,6 +529,14 @@ \subsection*{Positive selection for resistance alleles}
 Longer shared haplotypes indicate a more recent common ancestor, and thus some of these core haplotypes may have experienced more recent and/or more intense selection than others.
 %%
 
+%%
+As sample collections took place over 12 years (2000-2012), it might be expected that core haplotypes appearing earlier in our sampling would have smaller shared haplotypes due to increased opportunity for recombination and mutation.
+%
+However, no correlation was found between the year a core haplotype was first detected and the median length (r(8)=0.03, p=0.93,  Supplementary Figure \ref{fig:pspd}).
+%
+It should be noted that, due to the retrospective nature of this study, the detection date of a core haplotype (or haplotype group) does not signify its origin.
```

Comment:
This sentence isn't clear what you mean. Do you need it at all?

Reply:
Removed.